### ISSUE TITLE
LandingPage Missing ARIA Landmarks

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#777]
The only ARIA landmark region present on this page is the footer (“content info”).  No other regions are made available to screen reader users, who will find it difficult to obtain a high-level overview of how the page content is structured and navigate effectively. 
 
Suggested Solution 
 
All page content must be located inside an appropriate ARIA landmark region.  The chosen regions must be a good semantic fit with the content they contain.  For example: 
 
• Header (“banner” landmark region) wrapping the “Sign In” button, MathShare logo and “Help students …” introductory text”. 
• Main landmark region wrapping everything between the end of the header and start of the footer. 
 
Ideally, these regions should be implemented using the native semantic elements (< header>, < main>, < footer>, etc.).  If this is not possible, the appropriate role attribute should be added to < div> containers, but we would stress that native controls are always best and there shouldn’t be a reason to avoid them here.

**Comment by rupeshparab** [comment]
I have fixed the mentioned 4 points

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/688f59a0586164e923e6789b7f3e1e31e9216522.
